MATLAB/DFA/src/EDA/exper_data.m

```diff
@@ -0,0 +1,116 @@
+%{ 
+Experimental Data
+Author: Surabhi Joshi
+
+Table of Contents
+Experimental Data
+Summary
+Create tables for averages and standard deviations. 
+Create a corelation matrix and a following heatmap
+Create the correlation matrix, CV 
+Create heatmaps to visually understand the standard deviations and averages. 
+Scatterplots of the mean and standard deviations
+Hisograms of the mean and standard deviations
+%}
+
+%% Summary
+%This script reads in some experimental data and performs exploratory initial statistical analysis to understand how each condition affects the metabolomics before running DFA. 
+% download the experimental data and create two tables, one for each
+% cancer cell line TU8902 and HCT116.
+experimental_data_TU8902 = readtable('Barb Nelson Pos Table-20180410 - HL.xlsx', ...
+'Sheet', 'TU8902');
+experimental_data_HCT116 = readtable('Barb Nelson Pos Table-20180410 - HL.xlsx', ... 
+ 'Sheet',"HCT116");
+% delete row 2 and column 2 from both tables
+experimental_data_TU8902.transition = [];
+experimental_data_HCT116.transition = [];
+
+%download metabolomics and clusterring data
+metabolomics = readtable('Barb Nelson Pos Table-20180410 - HL.xlsx', ... 
+    'Sheet', 'Clustering');
+metabolomics(1,:) = [];
+
+%% Create tables for averages and standard deviations. 
+%This here is going to take the averages and standard deviations from the original experimental data and fill two new tables with their values. Any NaN will be turned to 0 prior to finding the mean and standard deviations. 
+Tabl_average_TU8902 = table_calc(experimental_data_TU8902, 'mean');
+Tabl_average_TU8902(227,:) = [];
+Tabl_std_TU8902 = table_calc(experimental_data_TU8902, 'stddev');

```

Comment:
I would suggest that you rename the function to something more descriptive, such as computeReplicateMetrics, just to make it clear it computes stuff like the mean, standard deviation, etc.